robots/perf-report-creator/scrape.sh

```
@@ -9,12 +11,24 @@ if [ -z "${KUBEVIRT_PROVIDER}" ]; then
     exit 1
 fi
 
+# The scrape window overlaps the previous ISO week so that late artifact
+# uploads and job start time drift are still picked up. Overlapping runs are
+# deduplicated when weekly-report merges into the published data.
+SINCE=${SINCE:-192h0s}
+
 # scrape sig-performance ${KUBEVIRT_PROVIDER} results
-perf-report-creator results --credentials-file=${CREDENTIALS_FILE} --output-dir ${OUTPUT_DIR}/results --since 168h0s --performance-job-name periodic-kubevirt-e2e-${KUBEVIRT_PROVIDER}-sig-performance
+perf-report-creator results --credentials-file=${CREDENTIALS_FILE} --output-dir ${OUTPUT_DIR}/results --since ${SINCE} --performance-job-name periodic-kubevirt-e2e-${KUBEVIRT_PROVIDER}-sig-performance
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)